backend/db/db.py

```diff
@@ -754,6 +754,12 @@ async def get_cached_change_points(
         data, meta = separate_meta_one(results[0])
 
         series_last_modified = series_id_tuple[3]
+        if not meta:
+            # Series doesn't have a last_modified field. It probably predates the time we even
+            # had caching for change points.
+            # Caller needs to compute the change_points now.
+            return {}
```

Comment:
I don't follow this. Wasn't this already fixed here? https://github.com/nyrkio/nyrkio/commit/429e4c697e50fb0ff689b2e4a75b716042e9a35c

Reply:
This happens earlier than that, so your fix doesn't help.

How this slipped by I don't know.